oedisi/tools/cli_tools.py

```diff
@@ -80,8 +102,183 @@ def build(target_directory, system, component_dict):
     )
     with open(f"{target_directory}/system_runner.json", "w") as f:
         f.write(runner_config.json(indent=2))
+    
+    
+    yaml_file_path = f"{target_directory}/docker-compose.yml"
+    
+    if multi_container:
+        edit_docker_files(wiring_diagram, target_directory)
+        create_docker_compose_file(wiring_diagram, target_directory)
+        clone_broker(target_directory, yaml_file_path)
+        create_kubernetes_deployment(target_directory, wiring_diagram)
+    
+def create_kubernetes_deployment(target_directory, wiring_diagram):
+    kube_folder = os.path.join(target_directory, "kubernetes")
+    if not os.path.exists(kube_folder):
+        os.mkdir(kube_folder)
+    
+    service = {
+        "apiVersion": "v1",
+        "kind": "Service",
+        "metadata": {
+            "name": f"{KUBERNETES_APP}-service",
+        },
+        "spec" : {
+            "type": "NodePort",
+            "selector" : {
+                "app": KUBERNETES_APP
+            },
+            "ports" : [
+                {
+                    "port": BROKER_PORT,
+                    "targetPort": BROKER_PORT
+                }
+            ]
+        }
+    }
```

Comment:
Does this only expose the broker?

Reply:
No, all containers expose a port.

See line 169 - 183